package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@moneytree/mt-link-javascript-sdk",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

We will update the version when we will actually publish the new SDK with a new pull request.

```suggestion
  "version": "3.0.0",
```